src/libstd/sync/once.rs

```diff
@@ -102,11 +102,11 @@ impl Once {
         // calling `call_once` will return immediately before the initialization
         // has completed.
 
-        let prev = self.cnt.fetch_add(1, Ordering::SeqCst);
+        let prev = self.cnt.fetch_add(1, Ordering::AcqRel);
```

Comment:
This can be `Relaxed` with a `fence(Acquire)` if the count is negative: since `swap` and `fetch_add` are atomic, no extra synchronization is necessary to make the number of threads that pass through the fetch_add seeing a positive integer exactly the number that will be read as the result of the `swap`.